src/temperature_component.cpp

```diff
@@ -226,6 +228,15 @@ void TemperatureComponent::prepareToRun() {
     // Constants & conversion factors
     kcon = secs_per_Year / 10000;               // conversion factor from cm2/s to m2/yr;
     ocean_area = (1.0 - flnd) * earth_area;    // m2
+    
+
+    aCO2 = core->sendMessage( M_GETDATA, D_ACO2 );
```

Comment:
This is how the ocean component reads in tgav 🤷‍♀️, but aCO2 ends up being 0 which causes q2co to be 0